test/e2e/toolchain_test.go

```diff
@@ -54,29 +57,33 @@ func TestToolchain(t *testing.T) {
 
 		err = await.WaitForCheInstallConditions(cheInstallation.Name, UntilHasCheStatusCondition(cheinstallation.InstallationSucceeded()))
 		require.NoError(t, err)
-		checkCheResources(t, f.Client.Client, cheOperatorNS, cheOg, cheSub, cheCluster)
+		cluster, err := await.GetCheCluster(cheCluster.Namespace, cheCluster.Name)
+		require.NoError(t, err)
```

Comment:
Why do you need to get the CheCluster resource - shouldn't that be loaded by the `CheClusterAssertion`?

Reply:
at this stage of the e2e test, the initial `cheCluster` obj is not up-to-date so I need to explicitly get it again in order to compare its `status.CheURL` with what's in the `CheInstallation` resource.

Reply:
Sure, but the [loadCheClusterAssertion()](https://github.com/codeready-toolchain/toolchain-operator/pull/62/files#diff-b11ae265569f1cc9752c7c327b74b47cR38) function should ensure that it loads the latest version of the resource. 
After all, you fixed this behavior a few days ago: https://github.com/codeready-toolchain/toolchain-operator/commit/221b8bef254238e8ca4d2bb154d20286aaa90208#diff-b11ae265569f1cc9752c7c327b74b47c

Reply:
good point. Let me try without this explicit loading.

Reply:
ok, so I removed this unnecessary loading in e9cfc39